fastify.d.ts

```diff
@@ -118,6 +118,20 @@ export type FastifyServerOptions<
 
 type TrustProxyFunction = (address: string, hop: number) => boolean
 
+declare module 'fastify-error' {
+  interface FastifyError {
+    validation?: ValidationResult[];
+  }
+}
+
+export interface ValidationResult {
+  keyword: string;
+  dataPath: string;
+  schemaPath: string;
+  params: Record<string, string | string[]>;
+  message: string;
+}
```

Comment:
Why is all of this needed?

Reply:
Not sure. it exists in fastify-error, but the actual validation is done in core so i'm moving the types over to here